index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+
+</head>
+<body>
+  <h1 class="title">Título del libro <span class="genre">(género)</span></h1>
+  <ul class="details">
```

Comment:
I think the details of the book are not really a list of some sort 🤔 . Maybe just a `div` is enough 😄